dags/tests/general/test_monitoring.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from airflow.models import Variable
+
+Variable.set("stitch_secret", "test_secret_value")
```

Comment:
This requires that airflow backend have been configured and initialized. There are two approaches that can be used:
- Set the variables as env vars before running the tests (i.e. `AIRFLOW_VAR_STITCH_SECRET=test_secret_value AIRFLOW_VAR_STITCH_LOADS_ENDPOINT=test_loads_endpoint AIRFLOW_VAR_STITCH_EXTRACTIONS_ENDPOINT=test_extractions_endpoint pytest dags`). However this will need to be documented, as well as update `Makefile` and build.
- Mock `Variable`. Sample code would look like:
```
def mock_get(key, *args, **kwargs):
    return { "stitch_secret": "test_secret_value", ...}.get(key)
    
def test...(load_data, mocker):
    mocker.patch.object(Variable, 'get', new=mock_get)
```
- The previous solution would not work as is because `Variable.get` is used in `monitoring.py` and will be called at module's loading time. Reason is that DAG creation is defined on module level. One improvement is to use the methods under test to a separate module (i.e. `_helpers.py`).
- A simpler combination of the above solutions would be to move methods under test to `_helpers.py` and then mock environment variables by adding them in `config_utils` at `conftest.py`.